operators/Makefile

```diff
@@ -38,7 +38,10 @@ endif
 IMG_SUFFIX ?= -$(subst _,,$(USER))
 IMG ?= $(REGISTRY)/$(REPOSITORY)/$(NAME)$(IMG_SUFFIX)
 TAG ?= $(shell git rev-parse --short --verify HEAD)
-OPERATOR_IMAGE ?= $(IMG):$(TAG)
+OPERATOR_IMAGE ?= $(IMG):$(VERSION)-$(TAG)
+ifeq ($(SNAPSHOT_RELEASE), false)
```

Comment:
We already have a boolean variable `RELEASE`. Maybe we could reuse it instead of adding a new one?

Reply:
I renamed `RELEASE` to `USE_ELASTIC_DOCKER_REGISTRY`. It will be a proper name for it.

Reply:
> Maybe we could reuse it instead of adding a new one?

We need a way to understand for release build what to use in the tag of Docker image. Because for official releasе we will simply get version from git tag and for snapshot we will need image tag with version and git commit hash or with another id